inference-engine/tests/functional/inference_engine/lp_transformations/move_fake_quantize_transformation.cpp

```diff
@@ -0,0 +1,327 @@
+    return out << "_" << values.multiChannels << "_" << values.actual << "_" << values.result;
+}
+
+typedef std::tuple <
+    ngraph::element::Type,
+    ngraph::PartialShape,
+    MoveFakeQuantizeTransformationTestValues
+> MoveFakeQuantizeTransformationParams;
+
+class MoveFakeQuantizeTransformation : public LayerTransformation, public testing::WithParamInterface<MoveFakeQuantizeTransformationParams> {
+public:
+    void SetUp() override {
+        const ngraph::element::Type precision = std::get<0>(GetParam());
+        const ngraph::PartialShape shape = std::get<1>(GetParam());
+        MoveFakeQuantizeTransformationTestValues testValues = std::get<2>(GetParam());
+
+        // dequantization output precision depends on input precision
+        // to avoid huge amount of tests cases let's define dequantization output precision as input precision
+        if (!testValues.actual.dequantizationBefore1.multiply.empty()) {
+            testValues.actual.dequantizationBefore1.multiply.outPrecision = precision;
+        }
+        if (!testValues.actual.dequantizationBefore2.multiply.empty()) {
+            testValues.actual.dequantizationBefore2.multiply.outPrecision = precision;
+        }
+        actualFunction = ngraph::builder::subgraph::MoveFakeQuantize::get(
+            precision,
+            shape,
+            testValues.actual.fakeQuantizeBefore1,
+            testValues.actual.convertBefore1,
+            testValues.actual.dequantizationBefore1,
+            testValues.actual.fakeQuantizeBefore2,
+            testValues.actual.convertBefore2,
+            testValues.actual.dequantizationBefore2,
+            testValues.actual.operation,
+            testValues.actual.fakeQuantizeAfter,
+            testValues.actual.convertAfter,
+            testValues.actual.dequantizationAfter,
+            ngraph::element::undefined,
+            {},
+            testValues.axis);
+        auto supportedPrecisionsOnActivation = std::vector<ngraph::pass::low_precision::OperationPrecisionRestriction>({
+                ngraph::pass::low_precision::OperationPrecisionRestriction::create<ngraph::opset1::AvgPool>({{0, testValues.params.precisionsOnActivations}})
+            });
+
+        auto quantizationRestrictions = testValues.multiChannels ?
+            std::vector<ngraph::pass::low_precision::OperationPerTensorQuantizationRestriction>() :
+            std::vector<ngraph::pass::low_precision::OperationPerTensorQuantizationRestriction>({
+                ngraph::pass::low_precision::OperationPerTensorQuantizationRestriction::create<ngraph::opset1::AvgPool>()
+                });
+
+        const auto params = TestTransformationParams::toParams(testValues.params);
+        SimpleLowPrecisionTransformer transformer(supportedPrecisionsOnActivation, quantizationRestrictions);
+        transformer.commonGraphRewrite->add_matcher<ngraph::pass::low_precision::MoveFakeQuantize>(params);
+        transformer.transform(actualFunction);
```

Comment:
Use implemented transformation only:
```suggestion
        ov::pass::Manager manager;
        manager.register_pass<ngraph::pass::low_precision::MoveFakeQuantize>(params);
        manager.run_passes(actualFunction);
```